lib/inherited_resources/base_helpers.rb

```diff
@@ -132,12 +132,12 @@ def association_chain
           end.compact.freeze
       end
 
-      # Overwrite this method to provide other interpolation options when
-      # the flash message is going to be set.
-      #
-      # def flash_interpolation_options
-      #    { }
-      # end
+    # Overwrite this method to provide other interpolation options when
+    # the flash message is going to be set.
+    #
+    # def flash_interpolation_options
+    #    { }
+    # end
```

Comment:
The indentation here was correct and it seems like a bug in rubocop that it's being changed. I'll revert these changes and report upstream.

Reply:
Sounds good thanks!

Reply:
Sounds good thanks!

Reply:
Rebased by still pending to remove these changes, I'll ping you when it's done!

Reply:
@javierjulio Done!

Reply:
Ok great thanks! 👍🏻 I'll take a look now.